Source/JavaScriptCore/jit/ThunkGenerators.cpp

```diff
@@ -425,36 +425,6 @@ MacroAssemblerCodeRef<JITThunkPtrTag> polymorphicThunkForTailCallForClosure(VM&
     return polymorphicThunkFor(vm, CallMode::Tail, ClosureMode::Yes);
 }
 
-MacroAssemblerCodeRef<JITThunkPtrTag> polymorphicRepatchThunk(VM&)
```

Comment:
I don't understand why this was entirely removed, as it seems it feeds a useful niche -- for my own elucidation, why is it no longer necessary?